webapi/CopilotChat/Controllers/ServiceOptionsController.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using SemanticKernel.Service.CopilotChat.Models;
+using SemanticKernel.Service.Options;
+
+namespace SemanticKernel.Service.CopilotChat.Controllers;
+
+/// <summary>
+/// Controller responsible for returning the service options to the client.
+/// </summary>
+[ApiController]
+[Authorize]
+public class ServiceOptionsController : ControllerBase
+{
+    private readonly ILogger<ServiceOptionsController> _logger;
+
+    private readonly MemoriesStoreOptions _memoriesStoreOptions;
+
+    public ServiceOptionsController(
+        ILogger<ServiceOptionsController> logger,
+        IOptions<MemoriesStoreOptions> memoriesStoreOptions)
+    {
+        this._logger = logger;
+        this._memoriesStoreOptions = memoriesStoreOptions.Value;
+    }
+
+    // TODO: [Issue #95] Include all service options in a single response.
```

Comment:
I'm not sure that we should create an issue for this since it's not a current problem. we want to use a single response whenever we add another option, but we don't know when that will be, and I don't think we should bloat the issues list with things that don't currently need a solution.